R/loo_approximate_posterior.R

```diff
@@ -178,9 +178,11 @@ loo_approximate_posterior.function <-
            log_p,
            log_g,
            r_eff = NULL,
-           save_psis = FALSE) {
+           save_psis = FALSE,
+           is_method) {
 
-    if(!is.null(r_eff)) warning("r_eff not implemented for aploo")
+    if(!is.null(r_eff)) warning("r_eff not implemented for aploo.")
```

Comment:
Just a minor thing. Tere and in general, please add spaces after `if` and before `(...)` according to general R code guidelines (same for all special function calls such as `for` etc.) Please also add a space after `(...)` and `{` in `if` and related statements.

Reply:
This should be something we include in a pre-commit (that I now learnt from Eero) and include in the project. Then it is easily enforced in the test suites as well. I could add this as an issue.

Reply:
Now fixed for all code